### PR TITLE
Bly 1152

### PR DESCRIPTION
**Pull request by FiboApe**

### CODE REVIEW

File: apps/root/src/services/web3Service.ts [hunk 1]
```diff
@@ -424,10 +414,9 @@ export default class Web3Service {
 
     this.setAccount('');
```
Comment (th0rOdinson) [inline apps/root/src/services/web3Service.ts:415]: I understand that handling the account here is useful for re-evaluating components, but perhaps removing `getAccount` method could prevent future confusion regarding where to obtain account data

**Review by th0rOdinson — APPROVED** [review]
This is freaking amazing! Hiding the essential and huge improve on UX 🚀 . 
Well done!

File: apps/root/src/services/web3Service.ts (hunk 1 above)
Comment (FiboApe) [reply apps/root/src/services/web3Service.ts:415]: Sure!

File: apps/root/src/hooks/useWalletNetwork.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useWallets } from '@privy-io/react-auth';
+import useAccountService from './useAccountService';
+import { Network } from '@ethersproject/providers';
+
+function useUnderlyingAmount(walletAddress: string): [Nullable<Network>, boolean, string?] {
```
Comment (th0rOdinson) [inline apps/root/src/hooks/useWalletNetwork.ts:6]: Should change to `useWalletNetwork`

File: apps/root/src/pages/dca/positions/components/current-positions/components/position-controls/index.tsx [hunk 3]
```diff
@@ -100,6 +106,7 @@ const PositionControls = ({
     return supportedNetwork;
   }, [chainId]);
 
+  // console.log(connectedNetwork);
```
Comment (th0rOdinson) [inline apps/root/src/pages/dca/positions/components/current-positions/components/position-controls/index.tsx:109]: Debugging evidence 👀

**Review by th0rOdinson — APPROVED** [review]
Fantastic job! Two minor comments. (All tests passed 💯 )

**Pull request merged**